### ISSUE TITLE
Flag for use of artistic controls

### ISSUE THREAD

**Issue by simontWork** [w3c/PNG-spec#344]
Do we need to follow the example of other SDOs and add a metadata flag indicating that no artistic control of chrominance, luminance, saturation etc. has been applied?

Whilst many contributors are from an entertainment/photography background where the object is to create pleasing images, one can also see that in some applications (e.g. medical imaging), one might like to be able to flag that no such artistic adjustment has been made.  That is, that the PNG image represents as closely as possible the colours and tonal ranges in the scene.

**Comment by svgeesus** [comment]
An interesting and useful suggestion, though I would want to engage with the medical imaging community first to establish their precise needs._

**Comment by ProgramMax** [comment]
Agreed. The idea makes sense. But I think we should have some pressure from industries before we standardize it. In fact, I think that is a piece of W3C's requirements.
